lib/ibmdb.js

```diff
@@ -409,32 +425,37 @@ IBMDB.prototype.updateOrCreate = IBMDB.prototype.save =
       // Execution for updateOrCreate requires running two
       // separate SQL statements.  The second depends on the
       // result of the first.
+      var where = {};
+      where[id] = data[id];
```

Comment:
You are right @amir, I realize this is from the `idName` now. We should leave it the the way proposed. :+1: